.circleci/config.yml

```diff
@@ -118,6 +118,11 @@ node-integration-base: &node-integration-base
   working_directory: ~/dd-trace-js
   steps:
     - checkout-and-yarn-install
+    - run:
+        name: Install Browser Dependencies
```

Comment:
Not all integration tests need those. Should this be an explicit browser test instead?

Reply:
we could i guess but likely it would mean this isn't checked as often

Reply:
Why would it be checked less often?